packages/stencil/src/components/datacom-avatar/useage.md

```diff
@@ -0,0 +1,11 @@
+# Avatar Component Usage
+
+The Avatar component is composed of one web component: 'datacom-avatar'.
+
+The component can be rendered as shown below, where the label content is passed as a prop on the component.
+
+### Example
+
+```html
+<datacom-toggle firstName="Sally" lastName="Mei"  jobTitle="UX/UI Designer" companyName="Datacom"></datacom-toggle>
```

Comment:
I'm assuming datacom-toggle is incorrect here, yeah? This should be your component instead.